base/shell/explorer/settings.cpp

```diff
@@ -60,6 +61,9 @@ BOOL TaskbarSettings::Load()
     cbSize = sizeof(sr);
     dwRet = SHGetValueW(hkExplorer, L"StuckRects2", L"Settings", NULL, &sr, &cbSize);
 
+    dwRet = SHGetValueW(hkExplorer, L"Advanced", L"TaskbarSmallIcons", NULL, &dwValue, &cbSize);
+    bSmallIcons = (dwRet == ERROR_SUCCESS) ? (dwValue != 0) : TRUE; /* This is an opt-out setting */
```

Comment:
If it uses small icons by default, this is rather an opt-in :)

Reply:
I removed the unneeded comment. Thanks :)